community/matrix.md

```diff
@@ -8,33 +8,42 @@ editor: markdown
 dateCreated: 2022-01-26T00:49:51.001Z
 ---
 
-We have a Matrix homeserver at `matrix.mathesar.org`, but you can join our public rooms using any Matrix account. If you haven't used Matrix before, we recommend getting started [using Element](https://element.io/get-started).
+For realtime chat, we use [Matrix](https://matrix.org/) *(similar to Slack, Discord, Mattermost, Zulip, etc)*.
 
-A list of our public rooms is below. You can also join our [Matrix Space](https://matrix.to/#/!KQLkDbtIMsvcwUIfNy:matrix.mathesar.org?via=matrix.mathesar.org&via=matrix.org) which contains all these rooms.
+## Joining
 
-## General discussion
+1. Join our [Matrix Space](https://matrix.to/#/!KQLkDbtIMsvcwUIfNy:matrix.mathesar.org?via=matrix.mathesar.org&via=matrix.org).
+
+    - If you're new to Matrix, you'll need to create an account. You can choose to host your account on `matrix.org` or on our homeserver `matrix.mathesar.org`.
+
+1. If you want to use a desktop or mobile app, we recommend installing [Element](https://element.io/get-started).
+
+
+### Chat rooms
```

Comment:
It might also be worth explaining that they'll need to join whichever chat rooms they're interested in individually, even if they have joined our Matrix space per the instructions above.

Reply:
These points should both be addressed in 05fa0e497fe3d54ea5ea9dd2bb3ca3c0ac3511a2